src/notation/view/widgets/editstyle.ui

```diff
@@ -5874,74 +5917,38 @@
-                    <property name="keyboardTracking">
-                     <bool>false</bool>
-                    </property>
-                    <property name="suffix">
-                     <string>sp</string>
-                    </property>
-                    <property name="minimum">
-                     <double>2.000000000000000</double>
-                    </property>
-                    <property name="value">
-                     <double>4.000000000000000</double>
-                    </property>
-                   </widget>
-                  </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Cap width after:</string>
                     </property>
                    </widget>
                   </item>
-                  <item row="2" column="1">
-                   <widget class="QSpinBox" name="mmRestRefDuration">
-                    <property name="suffix">
-                     <string> / 4</string>
-                    </property>
-                    <property name="minimum">
-                     <number>2</number>
-                    </property>
-                    <property name="maximum">
-                     <number>8</number>
+                  <item row="2" column="3">
+                   <widget class="QToolButton" name="resetMMRestRefDuration">
+                    <property name="text">
+                     <string notr="true"/>
                     </property>
                    </widget>
                   </item>
-                  <item row="4" column="2">
-                   <widget class="QToolButton" name="mmRestMaxMeasuresReset">
+                  <item row="2" column="0">
+                   <widget class="QRadioButton" name="mmRestWidthConstant">
                     <property name="text">
-                     <string notr="true"/>
+                     <string>Fixed, based on time signature of:</string>
                     </property>
                    </widget>
                   </item>
-                  <item row="4" column="1">
-                   <widget class="QSpinBox" name="mmRestMaxMeasures">
-                    <property name="suffix">
-                     <string> measures</string>
+                  <item row="4" column="2">
+                   <widget class="QLabel" name="label_2">
+                    <property name="text">
+                     <string>measures</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="2">
-                   <widget class="QToolButton" name="resetMMRestRefDuration">
+                   <widget class="QLabel" name="label_17">
                     <property name="text">
-                     <string notr="true"/>
+                     <string>/ 4</string>
```

Comment:
All the above stuff moves the suffixes " / 4" and " measure" to after their spinboxes and this aboides translatable strings with leading spaces. New look:
![image](https://github.com/user-attachments/assets/4c98e8b7-8682-4ac3-8268-032e1efffffd)

Reply:
actually "quarters", plural, as the min is 2.

Reply:
Now only 'measure(s)', moved the " / 4" back to suffics and made untranslateble